src/SprykerEco/Zed/AfterPay/Business/Payment/Mapper/OrderToRequestTransfer.php

```diff
@@ -168,6 +170,31 @@ protected function buildOrderWithItemsRequestTransfer(AfterPayCallTransfer $afte
 
         $orderRequestTransfer = $this->addExpensesToOrderRequestTransfer($orderRequestTransfer, $afterPayCallTransfer);
         $orderRequestTransfer = $this->addGiftcardItems($orderRequestTransfer, $afterPayCallTransfer);
+        $orderRequestTransfer = $this->addShippingFeeToOrderRequestTransfer($orderRequestTransfer, $afterPayCallTransfer);
+
+        return $orderRequestTransfer;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\AfterPayRequestOrderTransfer $orderRequestTransfer
+     * @param \Generated\Shared\Transfer\AfterPayCallTransfer $afterPayCallTransfer
+     *
+     * @return \Generated\Shared\Transfer\AfterPayRequestOrderTransfer
+     */
+    protected function addShippingFeeToOrderRequestTransfer(
+        AfterPayRequestOrderTransfer $orderRequestTransfer,
+        AfterPayCallTransfer $afterPayCallTransfer
+    ): AfterPayRequestOrderTransfer {
+        $amount = (string)$this->moneyFacade
```

Comment:
why do you use to string conversion for money amount?

Reply:
I just writed it by existing example. See:
`OrderToRequestTransfer->addGiftcardItems()` function